docs/upgrading-solidus/v4.6.mdx

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: -4.6
+---
+
+import MinimalRequirements from '@site/src/theme/MinimalRequirements';
+
+# Solidus v4.6 (2025-09-09)
+
+<MinimalRequirements ruby="3.1" rails="7.0" />
+
+We're excited to announce the release of Solidus v4.6.0, packed with improvements across all components of the framework. This release focuses on enhanced email handling, reverse charge VAT support, improved admin UI components, and better Rails compatibility.
+
+## 🚀 Key Highlights
+
+### Event-Driven Email System
+This release introduces a major refactor of the email system, moving from coupled mailer calls to a flexible event subscriber pattern:
+
+- **New Order Events**: Added comprehensive order lifecycle events for better extensibility
+- **Configurable Email Subscribers**: Email sending is now handled by configurable event subscribers, making it easier to customize or replace email behavior
+- **Separated Mailer Logic**: Order confirmation, shipping notifications, and cancellation emails are now handled by dedicated subscribers
+
+### Reverse Charge VAT Support
+Added comprehensive support for EU reverse charge VAT requirements:
+
+- **Store Configuration**: New reverse charge status field for stores
+- **Address Integration**: Reverse charge fields added to address handling
+- **API Support**: Full API coverage for reverse charge functionality
+
+### Enhanced Admin UI Components
+Significant improvements to the modern Solidus Admin interface:
+
+- **New Select Component**: Improved performance and UX for dropdown selections
+- **Alert Component**: New alert system for better user feedback
+- **Refactored Address Forms**: Cleaner, more maintainable address input components
+- **Table Improvements**: Fixed sorting functionality and better semantic markup
+
+## 📦 Component Updates
+
+### Solidus Core
+- **Migration Improvements**: All migrations can now be properly rolled back
+- **Rails Compatibility**: Better support for multiple Rails versions with proper migration inheritance
+- **Database Relations**: Enhanced addressbook with proper foreign keys and associations
```

Comment:
I think there were more improvements to relations than just the addressbook, no?